Wrong package of SurfaceView

There's no such class as `android.widget.SurfaceView` however there's `android.view.SurfaceView`. ViewInspector tries to import non-existing class using wrong package.
App fails to compile with this error:
```
/<path>/app/build/generated/source/view-inspector/armv7MultiDex/debug/<my_package>/view_inspector/ProbeProxy$android_widget_SurfaceView.java:8: error: cannot find symbol
import android.widget.SurfaceView;
                     ^
  symbol:   class SurfaceView
  location: package android.widget
/<path>/app/build/generated/source/view-inspector/armv7MultiDex/debug/<my_package>/view_inspector/ProbeProxy$android_widget_SurfaceView.java:9: error: cannot find symbol
public final class ProbeProxy$android_widget_SurfaceView extends SurfaceView
                                                                 ^
  symbol: class SurfaceView
warning: The following options were not recognized by any processor: '[kapt.kotlin.generated]'
2 errors
```

--- a/view-inspector-plugin/src/main/groovy/view_inspector/plugin/internal/LayoutResourceParser.groovy
+++ b/view-inspector-plugin/src/main/groovy/view_inspector/plugin/internal/LayoutResourceParser.groovy
@@ -59,6 +59,8 @@ class LayoutResourceParser {
     def className
     if (name == 'View') {
       className = "android.view.View"
+    } else if (name == 'SurfaceView') {
+      className = "android.view.SurfaceView"
     } else if (name == 'WebView') {
       className = "android.webkit.WebView"
     } else if (

